ls1intum/Artemis
# `Development`: Add documentation for plagiarism checks

### Checklist
#### General




### Motivation and Context


Documentation about Artemis' plagiarism checks features are missing.

### Description

Add documentation about all available plagiarism checks for students, tutors and instructors.

### Steps for Testing


1. Look at the new documentation about plagiarism checks: [Link ](https://artemis-platform--5381.org.readthedocs.build/en/5381/user/plagiarism-check/)to documentation.

### Review Progress





#### Code Review
